src/utils/layout/plugins/RepeatingChildrenPlugin.tsx

```diff
@@ -231,7 +231,7 @@ export class RepeatingChildrenPlugin<E extends ExternalConfig = typeof defaultCo
     const out: string[] = [];
     for (let i = 0; i < rows.length; i++) {
       const row = rows[i];
-      row && out.push(...row.itemIds);
+      row && out.push(...(row.itemIds ?? emptyArray));
```

Comment:
Also a nitpick, but you can also just check if its undefined before pushing instead

```suggestion
      row?.itemIds && out.push(...row.itemIds);
```